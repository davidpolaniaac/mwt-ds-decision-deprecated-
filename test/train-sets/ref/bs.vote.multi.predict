0.000000 0.000000 0.000000
0.041258 0.000000 0.165033
0.037014 0.000000 0.148058
0.194066 0.000000 0.281402
0.198188 0.000000 0.287569
0.053451 0.000000 0.155553
0.263224 0.108261 0.323972
0.530437 0.334371 0.579987
0.260590 0.138097 0.312658
0.623179 0.491994 0.802263
0.373667 0.345480 0.409764
0.407278 0.296701 0.578739
0.653687 0.214216 0.691503
0.698699 0.404077 0.779290
0.359127 0.194051 0.467086
0.471713 0.447621 0.658849
0.738153 0.497381 0.776563
0.679279 0.632748 0.699377
0.961625 0.846292 1.078837
0.371225 0.256639 0.588491
0.421031 0.406493 0.635240
1.006710 0.534231 1.386442
0.719106 0.568045 0.855449
0.441948 0.434818 0.715776
0.294118 0.254717 0.352146
0.844598 0.537119 1.118137
0.473915 0.451506 0.644534
0.765563 0.560665 1.027926
0.586175 0.470868 0.605953
0.719393 0.566171 0.843057
0.920620 0.820930 0.985729
0.665044 0.333400 0.729816
0.675535 0.292014 0.746684
1.032238 0.835582 1.149324
1.024561 0.978270 1.115874
0.649288 0.523596 0.813854
1.200626 0.861985 1.434497
0.755949 0.538207 1.034530
1.088240 0.838307 1.290410
1.174038 0.898986 1.384085
0.312739 0.259948 0.521159
0.663338 0.565942 0.801054
0.976732 0.716282 1.174261
0.727770 0.607306 0.868094
0.727240 0.630561 0.840764
0.654950 0.505442 0.868526
1.027363 0.895612 1.736374
1.124330 1.071656 1.540534
0.756880 0.378841 0.873587
0.716539 0.444882 0.818013
0.904136 0.655278 1.085361
0.873126 0.772557 1.057850
0.991222 0.876981 1.166861
0.931603 0.851731 1.075140
0.828658 0.719158 0.948137
1.250577 1.123501 1.517079
1.132961 0.871863 1.291581
0.743996 0.612837 0.923135
1.286028 0.981842 1.529981
1.017076 0.806800 1.211975
0.791622 0.638449 0.914454
1.072303 0.712496 1.265780
0.894335 0.701968 0.990293
0.752621 0.434853 0.786216
1.115650 0.901428 1.349504
0.931886 0.619762 1.089103
0.429735 0.370273 0.818540
1.210725 0.495578 1.288672
0.762366 0.527644 0.927179
0.792666 0.467431 0.842085
1.045320 0.593316 1.297979
0.870318 0.671109 1.149071
0.758269 0.558884 0.917177
0.474683 0.459957 1.020855
0.361627 0.264029 0.545274
0.232912 0.121569 0.419695
0.737090 0.430444 0.917727
0.700845 0.558016 0.838422
0.295638 0.193284 0.440836
0.771593 0.708393 0.924531
0.326927 0.134463 0.485917
0.389688 0.243330 0.558989
0.786669 0.104232 0.849344
0.615163 0.554639 0.683434
0.529094 0.334042 0.533729
0.313162 0.279631 0.379622
0.209675 0.102734 0.288614
0.710269 0.573906 0.886307
0.902589 0.468160 1.164194
0.173962 -0.003624 0.601272
0.232152 0.162286 0.314017
0.302917 0.122719 0.432552
0.713384 0.432279 0.902288
0.259212 0.204582 0.378572
0.399109 0.207140 0.483184
0.318218 0.238760 0.616861
0.456855 0.430436 0.662567
0.652397 0.592202 0.701785
0.391596 0.255310 0.558475
0.323791 0.080374 0.499227
0.858431 0.731459 1.042138
0.343347 0.217919 0.603396
0.181470 0.026271 0.584494
0.686651 0.623264 0.803858
0.226538 0.162409 0.530250
0.609706 0.518258 0.690469
0.624003 0.411061 0.745736
0.218381 0.036538 0.340745
0.335522 0.269086 0.387784
0.361287 0.164244 0.548558
0.748188 0.483396 0.984191
0.380583 0.326028 0.593281
0.395036 0.373174 0.442815
0.680671 0.574639 0.831625
0.355845 0.189635 0.480897
0.300361 0.143763 0.388238
0.330026 0.227446 0.527868
0.544840 0.223253 0.589372
0.375008 0.271011 0.414272
0.369323 0.255855 0.497019
0.443785 0.430396 0.584152
0.587475 0.564957 0.605265
0.414192 0.358504 0.750753
0.821283 0.561140 1.055806
0.750336 0.657542 0.811234
0.520409 0.301204 0.526818
0.559177 0.500536 0.637776
0.567977 0.173054 0.582644
0.704590 0.469879 0.863764
0.564493 0.290989 0.581895
0.332627 0.209859 0.471660
0.381421 0.301568 0.630369
0.341460 0.262337 0.508351
0.384948 0.290912 0.439717
0.470614 0.454644 0.528507
0.632011 0.387631 0.775304
0.775013 0.658072 0.884307
0.529595 0.394046 0.556557
0.117128 0.067347 0.546341
0.748720 0.457214 0.777604
0.776973 0.081713 0.990856
0.260475 0.123409 0.431178
0.708418 0.558038 0.839650
0.703038 0.509381 0.857204
0.839123 0.758660 0.923783
0.250598 0.203302 0.621093
0.614736 0.257793 0.701938
0.970753 0.684787 1.296125
0.653731 0.546581 0.715700
0.762925 0.609479 0.977906
0.416942 0.343596 0.613114
0.570565 0.453565 0.595525
0.802972 0.699039 0.872309
0.929376 0.802597 1.131345
0.826387 0.773064 0.882122
0.736109 0.679996 0.790733
1.065995 0.821519 1.206804
1.028360 0.876227 1.184705
0.745926 0.638430 0.860349
0.959212 0.892868 0.989506
0.773926 0.663904 0.914616
0.848480 0.611629 1.085364
1.199543 1.034016 1.378084
1.019247 0.575567 1.310108
0.641325 0.528280 0.752936
0.649535 0.387198 0.713847
1.053294 0.847409 1.384879
0.893681 0.710114 1.046122
0.711978 0.448540 0.882891
1.005567 0.501730 1.292727
0.925618 0.753089 1.088482
0.794748 0.428781 0.851308
0.657674 0.056200 0.677738
0.540277 0.465845 0.578409
0.696784 0.326192 0.758562
0.921590 0.344810 0.960967
0.248356 0.193242 0.613759
0.835868 0.640149 1.154629
0.611582 0.511056 0.717236
0.892474 0.387025 1.176268
0.798078 0.650396 0.912986
0.696405 0.375709 0.758377
0.628536 0.417755 0.670758
0.178543 0.034128 0.396980
0.693362 0.141161 0.701032
0.733965 0.612369 0.808358
0.791921 0.680022 0.936695
0.946703 0.710665 1.276183
0.632996 0.280899 0.678387
0.813514 0.685258 0.910630
0.817366 0.451684 1.083901
0.152423 -0.189737 0.464664
0.386907 0.335778 0.487427
0.400081 0.354272 0.608350
0.304230 0.216469 0.569561
0.798480 0.664424 0.937707
0.606261 0.528500 0.682091
0.628258 0.396934 0.676104
0.404161 0.314606 0.486035
0.662332 0.423501 0.815634
0.818897 0.599465 1.198681
0.666202 0.444006 0.745522
1.985713 0.217497 2.305237
0.125610 0.009720 0.610486
0.263122 0.132152 0.513686
1.700797 1.439295 1.736483
0.143696 0.098901 0.182703
0.081444 -0.019331 0.510164
0.589254 0.550836 2.560719
0.483529 0.453664 1.021923
0.687296 0.427721 0.816875
1.999566 0.341128 2.003870
0.333576 0.278227 0.848862
0.358556 0.303663 0.706136
1.278106 1.230327 1.625183
0.372869 0.328733 0.848609
1.059688 0.794979 1.330024
1.581804 0.205459 1.745439
0.991163 0.467915 1.162204
0.600031 0.305884 0.607911
0.670369 0.585154 1.564143
1.189527 1.012043 1.537921
0.119897 0.031999 0.738372
1.767518 0.634158 1.955908
-0.033833 -0.298638 0.177263
0.607826 0.189603 0.706815
2.650965 1.048168 2.650965
0.090496 -0.236488 0.408123
0.777506 0.576108 0.928061
0.830188 0.618888 1.045780
0.890355 0.768219 1.103604
0.832741 0.444926 1.019207
2.379592 2.307381 2.666956
0.261386 0.222740 0.650724
1.139853 0.851335 1.387224
1.065606 0.980378 2.346185
1.042430 0.524254 1.307210
0.663658 -0.255680 0.730805
2.040842 0.928889 2.563099
1.167433 1.106943 1.543448
0.291580 0.147832 0.499101
1.858196 1.713146 1.922755
-0.048627 -0.422890 0.276687
0.765078 0.137631 0.818028
0.483791 0.483791 1.990389
0.155678 0.130376 0.680534
1.031643 0.380894 1.364457
1.859544 1.520614 2.139831
1.044482 0.467767 1.301225
0.741385 0.556047 0.857015
1.910315 0.807443 2.242766
0.928425 0.422238 1.071977
2.244235 0.548377 2.387461
-0.090695 -0.235260 0.950785
1.182654 0.812233 1.445712
1.444045 1.371125 2.260124
0.110806 -0.102339 0.366287
0.553658 0.343520 2.032098
1.117246 0.880926 1.480993
0.910127 0.691308 1.124617
0.121522 0.071102 0.985121
0.282252 0.140805 1.390547
0.897418 0.812553 0.958269
0.124574 -0.043721 0.891116
-0.088406 -0.223033 1.078083
0.022728 -0.051314 1.159963
0.227696 0.018502 0.577511
-0.450637 -0.492692 1.194983
0.586632 0.515916 0.646594
0.898013 0.742872 1.031669
-0.059621 -0.172807 0.998196
1.146388 1.084413 1.247002
0.866282 0.738994 0.929806
0.363605 0.242825 0.946261
0.824964 0.246485 1.118194
1.030735 0.396957 1.106297
0.308725 0.099268 0.424030
0.898676 0.341410 1.104251
0.190457 0.084095 0.256575
0.934264 0.734654 1.226992
0.197564 0.093251 0.322649
0.149301 0.083004 0.597996
1.176604 0.830856 1.354751
0.911317 -0.444640 0.981413
0.174704 -0.012969 0.325183
0.209389 0.038056 0.466260
0.278221 0.235077 1.067551
0.996865 0.822835 1.163692
-0.138659 -0.307307 0.947464
0.800258 0.407984 0.981901
0.151097 0.105088 0.191917
0.811390 0.591428 0.925356
1.149435 0.912386 1.379315
0.180302 0.045923 0.242303
0.796838 -0.360054 0.803472
0.258541 0.076753 0.487522
1.065622 0.833233 1.334028
0.790808 0.129056 0.860489
1.067089 0.836070 1.270873
0.535018 0.393782 0.554894
0.268153 0.225144 1.224271
0.688196 0.551031 0.920157
0.335840 0.187117 0.678324
-0.159423 -0.275916 0.998000
0.261726 0.125318 0.426210
0.970155 0.882179 1.057088
0.029006 -0.128260 0.151221
0.031762 -0.131467 0.192410
1.445539 0.438792 2.542680
0.662771 0.339022 0.666365
0.780056 0.585576 0.959958
0.694304 0.662711 1.736787
0.117376 0.082317 0.579463
1.019339 0.941888 1.216104
0.361650 0.223651 0.483687
0.166604 -0.097679 0.541966
1.075595 0.526698 1.322495
0.792964 0.599471 0.930260
0.954097 0.212480 1.090623
0.372744 0.357562 0.683488
0.001521 -0.072195 0.532846
1.019652 0.921899 1.075981
1.801519 0.549564 1.968571
0.983007 0.762271 1.120742
0.948817 0.820842 1.069257
0.274059 0.007353 0.707822
0.059095 0.007147 0.594827
0.705996 -0.059258 0.754897
-0.091586 -0.178675 0.841736
0.688717 0.324514 0.764336
0.101419 -0.064208 0.215662
0.927422 0.420925 0.985930
0.287917 0.050083 0.452892
0.427354 0.348309 1.144155
1.943433 0.519452 2.069813
0.557508 0.019529 0.622346
0.865807 0.627421 1.114256
0.285120 0.190601 0.832675
0.999589 0.398156 1.261313
0.379301 0.258800 0.483903
0.707839 0.031918 0.748525
0.067450 -0.097232 0.190355
0.984697 0.493372 1.001214
1.519140 0.483948 1.637048
0.897009 0.673953 1.067169
0.741353 -0.026007 0.934958
1.023398 0.904239 1.243806
0.885280 0.824110 2.304790
0.181881 0.137077 0.781272
0.952772 0.803111 1.054000
0.887923 0.309516 0.951626
0.415054 0.413858 2.864440
0.776355 0.568128 0.924083
0.405226 0.350749 0.948557
0.792163 0.708602 2.655971
0.923824 0.683605 1.079867
1.124131 0.958679 1.203365
2.753869 2.486275 2.828902
0.930860 0.288297 1.169050
0.931771 0.833058 1.039771
0.918868 0.305114 1.133428
0.033264 -0.110090 0.659091
0.911511 -0.136350 1.224722
1.350122 0.193338 1.516059
0.333096 0.258221 0.711354
0.928202 0.562552 1.107565
1.169906 1.061759 1.296170
0.717712 0.395156 0.767103
0.761264 -0.726840 0.794076
1.016600 0.612856 1.229840
1.074109 0.870214 1.331301
0.718494 0.139298 0.796273
-0.111114 -0.236868 0.032399
0.359710 0.324254 0.820511
0.406847 0.340607 0.760077
0.167818 -0.221366 0.642247
0.099009 -0.598052 0.316288
0.941612 0.626963 1.123345
0.994357 0.648284 1.167557
0.732116 0.615433 0.812082
1.037307 0.951146 1.156022
0.601223 0.312134 0.710594
0.772487 0.610337 0.918649
0.105812 -0.035791 0.452970
0.007743 -0.101782 0.610013
-0.747494 -0.791304 0.636218
0.831676 0.679698 0.942629
0.841158 0.288574 1.229743
0.138056 -0.136277 0.574833
0.868028 0.508833 1.146723
0.877005 0.655831 1.035127
0.249460 -0.031817 0.435649
0.021957 -0.080079 0.113897
0.131226 -0.065801 0.639572
0.257020 0.099980 0.445024
0.959077 0.889922 1.031209
0.795333 0.647491 1.101244
0.096744 0.014271 0.156800
0.311273 0.159439 0.484418
0.768448 0.302375 0.897387
1.116391 1.081031 1.199342
0.715616 0.202734 0.856565
2.294589 0.339812 2.417229
0.101907 -0.106936 0.491367
0.109904 -0.080034 0.438243
2.119828 1.744303 2.270882
0.094219 -0.007449 0.252462
0.274327 0.078601 0.363655
2.761176 2.328100 2.989328
0.651709 0.434498 0.741918
0.280889 0.232496 0.601733
2.708845 0.342625 2.731194
0.589969 0.068349 0.673318
0.253005 0.185838 0.366195
1.863550 1.296891 2.146153
0.967704 0.892761 1.072770
1.025108 0.987497 1.049127
2.016170 1.706062 2.331514
0.162071 0.143616 1.117379
0.364209 0.076147 0.482104
2.206827 0.588055 2.283137
1.205637 0.995039 1.438469
0.169107 -0.042263 0.758128
2.084118 1.866143 2.414660
-0.134537 -0.291775 0.098181
0.283628 0.093025 0.396851
2.671059 2.434493 2.786772
0.002862 -0.151037 0.198047
0.811628 0.556628 0.920802
0.758194 0.581485 0.894279
1.074382 0.983617 1.200979
0.317836 0.184066 0.633886
2.712982 2.205104 2.909763
0.174165 0.050778 0.234949
0.890312 0.783287 0.998630
2.824542 2.254206 3.000000
0.943863 0.833646 1.149665
-0.143616 -0.171880 0.526677
2.120134 0.847912 2.631598
1.406892 1.313866 1.546921
0.078958 -0.004341 0.143046
2.038305 1.632723 2.221723
-0.036959 -0.228894 0.118567
0.835482 0.088834 0.860135
0.520264 0.514293 2.492078
0.291944 -0.040217 0.505429
0.680669 0.133565 0.834474
2.252482 1.793768 2.450782
0.942109 0.846066 1.021544
0.740616 0.644885 0.874056
2.274381 0.763899 2.692840
0.799471 0.557813 1.005416
2.833748 0.276801 2.979913
0.143723 0.004172 0.718604
1.424745 1.376880 1.904127
2.429116 2.380367 2.923159
-0.068226 -0.141535 -0.020126
2.601179 0.371392 2.613635
0.991182 0.852091 1.257032
0.859037 0.787321 0.963984
-0.328482 -0.463310 0.869703
0.147103 -0.126471 0.365185
0.896982 0.756597 1.026044
0.082181 -0.056285 0.176707
-0.104405 -0.729904 0.105145
-0.008914 -0.192329 0.642350
0.067173 0.018216 0.152067
-0.720082 -0.785591 0.691261
0.304873 0.233399 0.462416
0.907252 0.784114 1.021737
-0.243413 -0.360368 -0.088223
1.206274 1.142311 1.313959
0.859294 0.717487 1.050908
0.352646 0.285390 0.484370
0.780125 0.526287 0.980845
0.936487 0.550255 1.131724
0.084970 -0.000358 0.171991
0.212292 0.152881 0.829533
0.066715 -0.005759 0.158383
0.972959 0.703276 1.141608
0.003013 -0.225007 0.336938
0.098489 -0.007622 0.381598
1.082898 0.961659 1.576711
-0.675170 -0.714439 -0.434934
0.002941 -0.145412 0.168885
0.019604 -0.050687 0.114173
1.040310 0.436958 1.208459
0.979655 0.843157 1.062499
-0.152616 -0.757360 0.037304
0.749440 0.635665 0.898244
0.086675 -0.041465 0.153270
0.950670 0.871555 1.035236
1.114068 0.917454 1.304591
0.122623 -0.040445 0.326092
-0.223207 -0.727551 -0.120701
0.112344 0.053472 0.601203
1.140171 0.876527 1.395881
0.167175 0.136357 0.929924
1.108029 0.865032 1.332113
0.663734 0.491876 0.767683
0.162916 0.084244 0.214354
0.894413 0.631268 1.284191
0.040585 -0.018549 1.065585
-0.696689 -0.887425 -0.175965
0.140741 -0.015399 0.308129
1.123422 1.010721 1.280371
0.137036 -0.065664 0.241675
0.126925 -0.001636 0.277354
2.768973 0.444786 2.945367
0.686126 0.558017 0.764378
1.007400 0.757813 1.289041
2.017565 0.918353 2.354488
0.215374 0.130477 0.516690
1.155052 0.975055 1.281581
0.337970 0.290821 0.567054
0.182528 -0.027672 0.430186
1.726196 0.484094 1.775637
1.148404 0.956242 1.445309
0.946385 0.816529 1.083200
0.471457 0.448013 0.671826
0.115590 -0.022111 0.244937
1.078553 1.025617 1.182287
1.976812 0.633402 2.825060
1.120223 0.989935 1.260734
1.116332 1.026750 1.178187
0.745505 0.476786 1.011950
0.080612 -0.047676 0.132949
0.807175 0.304056 0.996979
0.195441 0.164144 0.710530
0.806633 0.391962 0.881798
0.104148 0.072250 0.185691
1.011009 0.933522 1.107188
0.144881 0.111519 0.549682
0.979781 0.916077 1.075142
1.915270 0.508768 2.854694
0.209131 0.186887 0.634426
1.085547 0.997211 1.145699
1.025565 0.904702 1.130843
1.133684 0.889082 1.331963
0.416315 0.345047 0.521201
0.325729 0.222160 0.452721
0.070444 -0.032865 0.164988
1.179891 1.100104 1.270337
1.907309 1.360059 2.173015
0.991456 0.878364 1.068048
0.750139 0.524773 0.908153
1.113194 0.916697 1.317240
1.785853 1.545921 2.654551
0.347062 0.242684 0.714910
1.062850 0.994405 1.139611
0.836183 0.144237 0.911305
2.957425 2.884569 3.000000
0.888774 0.644231 1.025839
0.773845 0.380085 1.022635
2.722567 2.383001 2.881700
0.894470 0.789394 0.957005
1.062806 0.971576 1.160997
2.865126 2.425505 2.953607
0.778385 0.575379 1.154130
0.973483 0.887880 1.024900
0.335427 0.285230 1.041253
0.059970 -0.158340 0.166777
0.981680 0.057619 1.124275
1.076926 0.158257 1.191980
0.244536 0.133091 0.634240
0.912409 0.602931 1.034091
0.992272 0.792930 1.094189
0.363231 0.316140 0.768163
-0.879565 -0.957332 -0.745123
1.088381 0.961031 1.168193
1.060598 0.941535 1.132468
0.132359 0.030088 0.189962
-0.008390 -0.140340 0.096641
0.826342 0.389097 0.900519
0.197284 0.066681 0.798181
0.008267 -0.277573 0.255976
-0.678412 -0.773447 0.185097
0.977234 0.734497 1.132766
1.079565 1.002565 1.166083
0.712194 0.670390 0.763246
1.110821 1.001861 1.243103
0.362992 0.290797 0.650596
0.877331 0.793791 0.989055
0.089579 -0.074787 0.336271
0.015810 -0.074658 0.133973
-0.787064 -0.862773 -0.621745
0.944526 0.847320 1.029697
0.628340 0.192357 0.682400
0.089755 -0.133584 0.290000
0.888768 0.473183 1.052054
0.943243 0.792653 1.087659
0.204254 -0.032790 0.423253
-0.002204 -0.091083 0.074520
-0.059353 -0.137861 -0.020267
0.077704 0.026626 0.155992
1.067207 0.935695 1.240415
0.955805 0.886110 1.103984
0.149289 -0.029820 0.316957
0.065287 0.032081 0.680008
0.759627 0.448572 0.873944
