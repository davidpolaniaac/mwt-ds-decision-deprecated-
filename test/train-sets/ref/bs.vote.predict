0.000000 0.000000 0.000000
0.000000 0.000000 0.165033
0.000000 0.000000 0.148058
0.000000 0.000000 0.056914
0.000000 0.000000 0.055368
0.000000 0.000000 0.108588
1.000000 0.060035 0.099882
1.000000 0.185384 0.214994
1.000000 0.076365 0.133533
1.000000 0.153715 0.229805
1.000000 0.151824 0.281808
1.000000 0.124296 0.267574
1.000000 0.052844 0.236524
1.000000 0.146848 0.297702
1.000000 0.095344 0.189743
1.000000 0.263104 0.362490
1.000000 0.188637 0.294482
1.000000 0.314464 0.363651
1.000000 0.391540 0.490716
1.000000 0.129572 0.261093
1.000000 0.069807 0.134286
1.000000 0.280419 0.559181
1.000000 0.203773 0.384707
1.000000 0.212224 0.261401
1.000000 0.110674 0.136587
1.000000 0.292162 0.489802
1.000000 0.138171 0.244588
1.000000 0.214734 0.363004
1.000000 0.244205 0.385861
1.000000 0.231640 0.377310
1.000000 0.346767 0.538611
1.000000 0.116643 0.256667
1.000000 0.135187 0.213753
1.000000 0.266950 0.357525
1.000000 0.326916 0.541221
1.000000 0.232655 0.516905
1.000000 0.223680 0.378183
1.000000 0.279042 0.426095
1.000000 0.358019 0.456708
1.000000 0.444014 0.563562
1.000000 0.208084 0.394655
1.000000 0.265202 0.434631
1.000000 0.330610 0.459317
1.000000 0.204864 0.334043
1.000000 0.264325 0.352090
1.000000 0.122604 0.278542
1.000000 0.337852 0.663552
1.000000 0.389132 0.541451
1.000000 0.065278 0.283866
1.000000 0.137934 0.193465
1.000000 0.223702 0.296520
1.000000 0.273999 0.323600
1.000000 0.339832 0.371714
1.000000 0.293943 0.427828
1.000000 0.427890 0.529465
1.000000 0.535167 0.703843
1.000000 0.393061 0.459777
1.000000 0.255470 0.342168
1.000000 0.382347 0.476401
1.000000 0.279962 0.380802
1.000000 0.266670 0.319516
1.000000 0.354260 0.441986
1.000000 0.400923 0.619577
1.000000 0.178755 0.332358
1.000000 0.434999 0.491165
1.000000 0.225289 0.315372
1.000000 0.300479 0.432123
1.000000 0.609933 0.803870
1.000000 0.305149 0.435862
1.000000 0.348280 0.558524
1.000000 0.382903 0.416826
1.000000 0.409965 0.560847
1.000000 0.181245 0.388152
1.000000 0.206158 0.387712
1.000000 0.206952 0.514328
1.000000 0.258637 0.408113
1.000000 0.175173 0.466962
1.000000 0.328643 0.587575
1.000000 0.125153 0.153853
1.000000 0.140399 0.238694
1.000000 0.210751 0.259890
1.000000 0.220530 0.347812
1.000000 0.537810 0.595470
1.000000 0.246706 0.371924
1.000000 0.167971 0.435135
1.000000 0.231589 0.307771
1.000000 0.214186 0.235897
1.000000 0.368226 0.654590
1.000000 0.539060 0.683148
1.000000 0.200537 0.395830
1.000000 0.072891 0.139037
1.000000 0.255444 0.503878
1.000000 0.718733 0.908420
1.000000 0.148134 0.234985
1.000000 0.419400 0.526940
1.000000 0.281590 0.449685
1.000000 0.416415 0.524149
1.000000 0.181038 0.257179
1.000000 0.288000 0.435358
1.000000 0.241666 0.383677
1.000000 0.542362 0.815245
1.000000 0.406909 0.555072
1.000000 0.307712 0.434762
1.000000 0.413690 0.456131
1.000000 0.394815 0.766386
1.000000 0.465606 0.558527
1.000000 0.123853 0.193798
1.000000 0.322638 0.425156
1.000000 0.152502 0.277716
1.000000 0.302211 0.603111
1.000000 0.519083 0.691395
1.000000 0.299794 0.647601
1.000000 0.384462 0.517525
1.000000 0.221309 0.305192
1.000000 0.367274 0.503765
1.000000 0.337746 0.591077
1.000000 0.145899 0.263625
1.000000 0.414074 0.599592
1.000000 0.152904 0.557253
1.000000 0.345800 0.410094
1.000000 0.283921 0.442343
1.000000 0.420802 0.540704
1.000000 0.382728 0.467407
1.000000 0.332877 0.487905
1.000000 0.483052 0.669946
1.000000 0.311442 0.509110
1.000000 0.304542 0.405826
1.000000 0.476059 0.722197
1.000000 0.087009 0.356681
1.000000 0.286310 0.554956
1.000000 0.262545 0.395812
1.000000 0.456271 0.832506
1.000000 0.162775 0.234764
1.000000 0.237567 0.308066
1.000000 0.304797 0.417123
1.000000 0.187232 0.322590
1.000000 0.509059 0.586988
1.000000 0.337405 0.492913
1.000000 0.637149 0.806485
1.000000 0.290388 0.402846
1.000000 0.409257 0.463217
1.000000 0.205731 0.290107
1.000000 0.703565 0.780184
1.000000 0.762319 0.883820
1.000000 0.430969 0.536015
1.000000 0.453805 0.627823
1.000000 0.421150 0.595646
1.000000 0.778521 0.896804
1.000000 0.263792 0.487996
1.000000 0.314232 0.442666
1.000000 0.340442 0.744408
1.000000 0.265696 0.386747
1.000000 0.588076 0.721545
1.000000 0.476270 0.573887
1.000000 0.537259 0.752284
1.000000 0.540044 0.642390
1.000000 0.789948 0.897156
1.000000 0.499521 0.624615
1.000000 0.674253 0.872025
1.000000 0.439344 0.610095
1.000000 0.407932 0.684588
1.000000 0.227526 0.417341
1.000000 0.245464 0.601820
1.000000 0.647146 0.707758
1.000000 0.292232 0.355189
1.000000 0.394226 0.478339
1.000000 0.729623 0.905942
1.000000 0.326497 0.618217
1.000000 0.424472 0.527758
1.000000 0.398577 0.780571
1.000000 0.644528 0.725458
1.000000 0.263530 0.378992
1.000000 0.215014 0.308513
1.000000 0.317181 0.408162
1.000000 0.124466 0.178991
1.000000 0.467242 0.638149
1.000000 0.281432 0.327047
1.000000 0.779583 0.985479
1.000000 0.539432 0.687287
1.000000 0.698828 0.777052
1.000000 0.396851 0.521240
1.000000 0.361413 0.418624
1.000000 0.443890 0.590839
1.000000 0.278819 0.443287
1.000000 0.163261 0.431669
1.000000 0.279712 0.564609
1.000000 0.600836 0.827837
1.000000 0.534030 0.688263
1.000000 0.198917 0.376790
1.000000 0.510130 0.786448
1.000000 0.450645 0.712192
1.000000 0.088898 0.485186
1.000000 0.209145 0.319710
1.000000 0.260771 0.315616
1.000000 0.256404 0.383797
1.000000 0.454846 0.706750
1.000000 0.446069 0.578452
1.000000 0.282540 0.459406
1.000000 0.254622 0.458421
1.000000 0.386176 0.671750
