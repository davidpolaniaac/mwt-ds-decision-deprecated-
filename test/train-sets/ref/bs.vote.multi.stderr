final_regressor = models/bs.vote.multi.model
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
decay_learning_rate = 1
predictions = bs.vote.multi.predict
creating cache_file = train-sets/0001.multi.dat.cache
Reading datafile = train-sets/0001.multi.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
1.000000   1.000000            1         1.0   1.0000   0.0000       51
0.500851   0.001702            2         2.0   0.0000   0.0413      104
2.454662   4.408473            4         4.0   0.0000   0.1941      135
2.361341   2.268020            8         8.0   0.0000   0.5304      146
2.593392   2.825443           16        16.0   1.0000   0.7382      143
2.386648   2.179905           32        32.0   1.0000   1.0246       70
2.225134   2.063619           64        64.0  -1.0000   1.0453       34
1.582231   0.939329          128       128.0   0.0000   0.2605       30
1.210780   1.210780          256       256.0  -1.0000   0.9113       72 h
1.291089   1.371398          512       512.0   0.0000   0.3632       37 h

finished run
number of examples per pass = 180
passes used = 3
weighted example sum = 540
weighted label sum = 0
average loss = 1.17046 h
best constant = 0
total feature number = 41349
